CHANGELOG.md

```diff
@@ -2,9 +2,11 @@
 
 All notable changes to this project will be documented in this file.
 
-## [Unreleased]
```

Comment:
I think normally we keep unreleased as empty, don't we?

Reply:
we will push a new branch with the unrelease version in package.json